src/utils/wrapping.ts

```diff
@@ -12,15 +12,15 @@ export type FormState = {
 }
 
 export const aavePlugins = collateralPlugins.filter(
-  (p) => p.rewardToken === STAKE_AAVE_ADDRESS[CHAIN_ID]
+  (p) => p.rewardToken[0] === STAKE_AAVE_ADDRESS[CHAIN_ID]
```

Comment:
Let's change this condition to be `p.rewardToken.indexOf(STAKE_AAVE_ADDRESS[CHAIN_ID]) !== -1` so its agnostic of the address index